source/MRViewer/ImGuiMenu.h

```diff
@@ -259,14 +259,18 @@ class MRVIEWER_CLASS ImGuiMenu : public MR::ViewerPlugin,
   // override this to have custom UI in "Selection Properties" window (under "Draw Options")
 
   // override this to customize prefix for objects in scene
+  // \detail height should be less or equal ImGui::GetFrameHeight()
+  // method should save ImGui::CursorPosY
   MRVIEWER_API virtual void drawCustomObjectPrefixInScene_( const Object& )
   {}
   // override this to customize appearance of collapsing headers
   MRVIEWER_API virtual bool drawCollapsingHeader_( const char* label, ImGuiTreeNodeFlags flags = 0);
   // override this to customize appearance of collapsing headers for transform block
   MRVIEWER_API virtual bool drawCollapsingHeaderTransform_();
   // override this to have custom UI in "Scene" window (under opened(expanded) object line)
-  MRVIEWER_API virtual void draw_custom_tree_object_properties( Object& obj );
+  // \detail if onlyHeight is true, should return drawing height without rendering
+  // return 0.f if nothing drawing
+  MRVIEWER_API virtual float draw_custom_tree_object_properties( Object& obj, bool onlyCalcHeight );
```

Comment:
if we change signature lets rename function to camelCase

Reply:
fixed